### PR TITLE
local runs for linux too

### PR DESCRIPTION
**Pull request by heimojuh**
Hope you don't mind my bold approach with straight pull request. Just a minor tweak, added shell commands for starting local browsers in GNU/Linux. Verified in Debian and OSX. Couldn't get grunt to run tests. Thanks for the kick-ass tool!

### CODE REVIEW

File: lib/localbrowsers.js [hunk 1]
```diff
@@ -151,41 +156,66 @@ function LocalBrowsers(config, port) {
         } catch(e){}
 
         if(activeBrowsers.isActive) {
+        console.log(process.platform);
```
Comment (ryanseddon) [inline lib/localbrowsers.js]: Remove this please

File: lib/localbrowsers.js [hunk 2]
```diff
@@ -151,41 +156,66 @@ function LocalBrowsers(config, port) {
...
+                        if(activeBrowser !== "phantomjs") {
                             exec("osascript -e 'tell application \"" + activeBrowsers[activeBrowser].name + "\" to quit'");
                         } else {
                             activeBrowsers[activeBrowser].process.kill('SIGHUP');
                         }
                     }
+                    else if (process.platform === "linux") {
+                        if(activeBrowser !== "phantomjs") {
+
+                        activeBrowsers[activeBrowser].process.kill('SIGHUP');
```
Comment (ryanseddon) [inline lib/localbrowsers.js:175]: Can you fix up the formatting

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "bunyip",
   "description": "Automate client-side unit testing in real browsers using the cli",
-  "version": "0.2.0",
+  "version": "0.2.1-JH",
```
Comment (ryanseddon) [inline package.json]: Can you remove your initials from the version. I'd be happy to put a contributors list somewhere so you get credit for your work.

**Comment by heimojuh** [comment]
Ach, tired eyes. Sorry about the fumbles, the linux repo seems to miss one commit :(

**Comment by heimojuh** [comment]
There. And no contribution of any kind is necessary if you decide to accept. And sorry for the hassle!

**Comment by ryanseddon** [comment]
Thanks for contributing @heimojuh :thumbsup:

**Pull request closed without merging**